doc/introduction.rst

```diff
@@ -6,42 +6,56 @@ Introduction
 
 FPGA Synthesis
 --------------
 
-FPGA synthesis is supported for Ibex when the flip-flop based register file is used. Since latches are not well supported on FPGAs, it is crucial to select the flip-flop based register file.
+FPGA synthesis is supported for Ibex when the flip-flop based register file is used.
+Since latches are not well supported on FPGAs, it is crucial to select the flip-flop based register file.
 
-Outline
--------
+Contents
+--------
+
+:ref:`core-integration` provides the instantiation template and gives descriptions of the design parameters as well as the input and output ports.
+The instruction and data interfaces of Ibex are explained in :ref:`instruction-fetch` and :ref:`load-store-unit`, respectively.
+The two register-file flavors are described in :ref:`register-file`.
+The control and status registers are explained in :ref:`cs-registers`.
+:ref:`performance-counters` gives an overview of the performance monitors and event counters available in Ibex.
+:ref:`exceptions-interrupts` deals with the infrastructure for handling exceptions and interrupts,
+:ref:`debug-support` gives a brief overview on the debug infrastructure.
+For information regarding formal verification support, check out :ref:`rvfi`.
 
-This document summarizes all the functionality of the Ibex core in more detail. First, the instruction and data interfaces are explained in :ref:`instruction-fetch` and :ref:`load-store-unit`. :ref:`register-file` explains the register file. Control and status registers are explained in :ref:`cs-registers` and Chapter :ref:`performance-counters` gives an overview of all performance counters. Chapter :ref:`exceptions-interrupts` deals with exceptions and interrupts, and finally Chapter :ref:`debug-unit` summarizes the accessible debug registers.
 
 History
 -------
 
-Ibex development started in 2015 under the name "ZERO-RISCY" as part of the `PULP platform <http://pulp.ethz.ch/>`_ for
-energy-efficient computing.
-Much of the code was developed by simplifying the RV32 CPU core "RI5CY" to show how small a RISC-V CPU core could be.
-To make it even smaller, support for the ``E`` exension was added under the code name "MICRO-RISCY".
-Within PULP the core is used as the control core for PULP and PULPino.
+Ibex development started in 2015 under the name "Zero-riscy" as part of the `PULP platform <https://pulp-platform.org>`_ for energy-efficient computing.
+Much of the code was developed by simplifying the RV32 CPU core "RI5CY" to demonstrate how small a RISC-V CPU core could actually be.
+To make it even smaller, support for the "E" exension was added under the code name "Micro-riscy".
+In the PULP ecosystem, the core is used as the control core for PULP, PULPino and PULPissimo.
 
-In December 2018 lowRISC took over the development of ZERO-RISCY and renamed it to Ibex.
+In December 2018 lowRISC took over the development of Zero-riscy and renamed it to Ibex.
```

Comment:
Side note: It was previously written ZERO-RISCY (not consistently, but in most places). (No need to change it back, though.)

Reply:
Looking at the original paper, it's "Zero-riscy" there. So maybe that's the most official option then :-)

Reply:
Yep, exaclty. That's why I corrected it in the first place ;-)